Dockerfile

```diff
@@ -0,0 +1,13 @@
+# https://docs.docker.com/compose/rails/
+FROM ruby:2.2.3
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev
+
```

Comment:
Totally optional, but at WP we've added these lines to slim down the container size and remove some unnecessary cruft:
```docker
RUN apt-get clean
RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```